CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## `alpha` to `canary`
 - **Added:**
   - **Props:**
+    - `className`
```

Comment:
Do we want to support the style prop?

Reply:
Let's add it if there's demand for it, see my opening comment.